modules/products/bitbucket/helm.tf

```diff
@@ -6,7 +6,7 @@ resource "helm_release" "bitbucket" {
   repository = local.helm_chart_repository
   chart      = local.product_name
   version    = local.bitbucket_helm_chart_version
-  timeout    = 10 * 60 # autoscaler potentially needs to scale up the cluster
+  timeout    = 15 * 60 # autoscaler potentially needs to scale up the cluster
```

Comment:
Does Bitbucket Helm release need to depends_on pre-install job?

Reply:
When I test it, works without dependency.